sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -455,7 +455,7 @@ def list_entities(
         """
         user_select = kwargs.pop("select", None)
         if user_select and not isinstance(user_select, str):
-            user_select = ", ".join(user_select)
+            user_select = ",".join(user_select)
```

Comment:
Removed the space because cosmos will keep the space there on the returned object, which makes the deserialized object have a parameter with a preceding space. ie. ` value` instead of `value`.